dts/bindings/mfd/nordic,npm1300.yaml

```diff
@@ -18,3 +18,27 @@ properties:
   pmic-int-pin:
     type: int
     description: Pmic pin number for interrupt output
+
+  ship-to-active-time:
+    type: int
+    description: |
+      Press duration required to wake from ship / hibernate.
```

Comment:
Perhaps it would be good to mention that the duration is in milliseconds.

Reply:
if in ms, then add property suffix `-ms`